applications/callflow/src/module/cf_voicemail.erl

```diff
@@ -305,6 +305,7 @@ compose_voicemail(#mailbox{max_message_count=MaxCount
                                                andalso MaxCount > 0 ->
     lager:debug("voicemail box is full, cannot hold more messages, sending notification"),
     Props = [{<<"Account-DB">>, whapps_call:account_db(Call)}
+             ,{<<"Account-ID">>, whapps_call:account_id(Call)}
```

Comment:
Why not only put <<"Account-ID">> and use wh_util:format_account_id (or the other way around) ?